js_modules/dagster-ui/packages/ui-core/src/ticks/SensorDryRunDialog.tsx

```diff
@@ -76,12 +76,13 @@ const SensorDryRun = ({repoAddress, name, currentCursor, onClose, jobName}: Prop
   const [cursor, setCursor] = useState(currentCursor);
 
   const [submitting, setSubmitting] = useState(false);
+  const [launching, setLaunching] = useState(false);
   const [error, setError] = useState<PythonErrorFragment | null>(null);
-  const [sensorExecutionData, setSensorExecutionData] = useState<DryRunInstigationTick | null>(
-    null,
-  );
+  const [sensorExecutionData, setSensorExecutionData] =
+    useState<SensorDryRunInstigationTick | null>(null);
+  const [executionParamsList, setExecutionParamsList] = useState<ExecutionParams[]>([]);
 
```

Comment:
```
const executionParamsList = useMemo(() => buildExecutionParamsListSensor(data, sensorSelector), [sensorSelector, data])
```

Reply:
Just committed to address this: https://github.com/dagster-io/dagster/pull/25633/commits/fd6a449865aa63775653674f332a892efb8b4e18

Not sure if it's how you wanted it, feel free to take a look

Reply:
https://github.com/dagster-io/dagster/pull/25633/files#diff-515db7eaf248b881449756acdc385f71173633288183df1f034c7c968657fc52R97-R104